test/common/http/http2/http2_connection.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package test.common.http.http2;
+import "test/fuzz/common.proto";
+
+message Http2DataFrame {
+  bytes data = 1;
+  // Use only 255 bytes maximum
+  bytes padding = 2;
+}
+
+message Http2HeaderFrame {
+  uint32 stream_dependency = 1;
+  uint32 weigth = 2;
```

Comment:
spelling: weight?

Reply:
Fixed the spelling, also the field was not in use, so I fixed that too.